metricbeat/docs/metricbeat-options.asciidoc

```diff
@@ -207,6 +207,18 @@ processors in your config.
 The modules and metricsets for which the host is defined as a HTTP URL, also
 support the following options:
 
+[float]
```

Comment:
I wonder why you put `headers` on the top? Alphabetical? Form a priority perspective I would probably put it after username / password.

Reply:
Ah, I was thinking they are more important, but I guess that's a bit of personal preference :) I see your point about username / password being more commonly-used options so they should come first. Will move `headers` below them.

Reply:
Whatever your preference is, let me know.